### PR TITLE
PDS 961 update documentation for timeoff

### PR DESCRIPTION
**Pull request by mkreitzerhumi**
Added time off endpoints and split out the individual endpoint documentation to separate files.

See https://github.com/Humi-HR/partners-api/tree/PDS-961-update-documentation-for-timeoff

### CODE REVIEW

File: v1/employees/get.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+# Employee List
```
Comment (thomas-humi) [inline v1/employees/get.md:1]: Any difference between "list" and "index"?
I've always used index, but list makes sense to me too.

File: v1/timeoff/employeeId/get.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+**Method** : `GET`
+
+**Auth required** : YES, include your token in the `Authorization` header. Ex: `Authorization: Bearer your-token-here`
+
+## Requests
+Requests must follow the [JSON:API](https://jsonapi.org/) spec.
+
+The `dateRange[start]` and `dateRange[end]` parameters are required.
+
+The results will onlyinclude time off requests where the start date(`start_at`), end date(`end_at`), or dates between
```
Comment (thomas-humi) [inline v1/timeoff/employeeId/get.md:16]:
```suggestion
The results will only include time off requests where the start date(`start_at`), end date(`end_at`), or dates between
```

**Review by thomas-humi — COMMENTED** [review]
Looks good!

A few comments, most of which we don't need to address in this version of the API.

File: v1/employees/get.md (hunk 1 above)
Comment (mkreitzerhumi) [reply v1/employees/get.md:1]: Nope, I'm only using it in an attempt to be more descriptive for the end user.

**Pull request merged**